scss/_base_code.scss

```diff
@@ -0,0 +1,28 @@
+// Base code styles
+@mixin vf-b-code {
+  code,
+  samp,
+  kbd {
+    font-family: $font-monospace;
+    text-align: left;
+  }
+
+  pre,
+  code {
+    text-align: left;
+    white-space: pre-line;
+    word-spacing: normal;
+    word-wrap: break-word;
+    tab-size: 4;
+    hyphens: none;
+    direction: ltr;
+  }
+
+  pre {
+    background: $color-dark;
+    color: $color-light;
+    border: 1px solid $color-mid-grey;
+    padding: 1rem;
+    border-radius: .5rem;
```

Comment:
Yes - if a user was to increase the font size, we would want the border radius to scale proportionately. The primary purpose of using rem units should be to ensure that whatever default font size a user has their browser set to, the layout will adjust to accommodate the size of text within it.